content/_index.md

```diff
@@ -3,25 +3,40 @@ title: Kubernetes Contributors
 description: Welcome to the Kubernetes community!
 ---
 
-This is the starting point for joining and contributing to the Kubernetes
-community---improving docs, improving code, giving talks, etc. To learn more
-about the project structure and organization, please refer to Project Governance
-information.
 
-## Communicating
+This site is the home of contributor documentation, and serves as a starting point for your journey as a Kubernetes contributor.
 
-The communication page lists communication channels like chat, issues, mailing
-lists, conferences, etc.
+This site is to serve as the hub for all things related to the Kubernetes Contributor experience. Who exactly is a contributor? We all are--Whether you’re writing docs, reviewing code, participating in the community and its many SIGs, everyone is welcome. We hope this site will be a pathway to success for our 35000+ Kubernetes contributors, providing current, up-to-date information on community events, contributor resources, Kubernetes SIGs, and more.
```

Comment:
Hm....I would not consider this a blocker but after re-reading this a few times this line and previous one are very similar. Maybe instead of `This site is to serve as` be `It is intended to be` or something along those lines?

Reply:
Have updated verbiage to reflect this suggestion!